src/rust/include/rust/orchard/incremental_sinsemilla_tree.h

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) 2020 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+#ifndef ZCASH_RUST_INCLUDE_INCREMENTAL_MERKLE_H
+#define ZCASH_RUST_INCLUDE_INCREMENTAL_MERKLE_H
```

Comment:
This guard is named incorrectly. Run `./test/lint/lint-include-guards.sh` to see the correct name.